batch/test/test_dag.py

```diff
@@ -48,7 +48,7 @@ def test_simple(client):
 def test_missing_parent_is_400(client):
     try:
         batch = client.create_batch()
-        batch.create_job('alpine:3.8', command=['echo', 'head'], parent_ids=[100000])
+        batch.create_job('alpine:3.8', command=['echo', 'head'], parent_ids=[(batch.id, 100000)])
```

Comment:
Parent_ids doesn't actually seem to use the batch.id:

`parent_ids = [pid[1] for pid in parent_ids]`

Do we need to pass the tuple?

Reply:
This will be fixed in my next PR.